Web/Extensions/WebHostBuilderExtensions.cs

```diff
@@ -57,15 +57,15 @@ public static IWebHostBuilder UseSerilogElasticSearchIngest(this IWebHostBuilder
             var options = new SiteOptions();
             context.Configuration.Bind(options);
 
-            if (options.Humio.IngestToken.IsSet() && options.Humio.IngestUrl.IsSet())
-                configuration.WriteTo.Elasticsearch(new ElasticsearchSinkOptions(new Uri(options.Humio.IngestUrl))
+            if (options.ElasticSearch.IngestToken.IsSet() && options.ElasticSearch.IngestUrl.IsSet())
+                configuration.WriteTo.Elasticsearch(new ElasticsearchSinkOptions(new Uri(options.ElasticSearch.IngestUrl))
```

Comment:
let's add a reminder somewhere to add these to keyvault

Reply:
I added a comment on the ticket. There will be many other things to fix for sure.